src/relayclient/DevRelayClient.ts

```diff
@@ -208,19 +207,19 @@ export class DevRelayClient extends RelayClient {
 
     const stakeManager = await IStakeManagerContract.at(stakeManagerAddress)
 
-    await stakeManager.stakeForAddress(relayServer.address, weekInSec, {
-      from: this.devConfig.relayOwner,
-      value: ether('1')
-      // gas: estim
-    })
-    this.debug('== sending balance to relayServer', relayServer.address)
-    await stakeManager.authorizeHub(relayServer.address, this.config.relayHubAddress, { from: this.devConfig.relayOwner })
+    await stakeManager.contract.methods.stakeForAddress(relayServer.getManagerAddress(), unstakeDelay).send({ from: this.devConfig.relayOwner, value: ether('1') })
```

Comment:
strangely, calling TruffleContract's stakeForAddress crashed. solved by using web3 calling

Reply:
`Number can only safely store up to 53 bits`
wasted quite some time to understand why (there are no numbers in the request.. didn't understand where it got there)

Reply:
It probably means that one of the parameters is a string instead of number, and it tries to add a number to it and it ends up being a concatenation. These kind of errors happened to me as well.